core/src/main/java/hudson/AboutJenkins.java

```diff
@@ -10,7 +12,7 @@
  * 
  * @author Kohsuke Kawaguchi
  */
-@Extension
+@Extension @Symbol("about")
```

Comment:
pipeline plugin only cares about `Descriptors`, but system-config-dsl plugin cares more than that. I think it makes sense to make one sweep to put them everywhere than finding out later that those annotations are missing on things one needs, especially given the long cycle time it takes for a change to get into the core and become usable in, say, LTS.